src/sharedHelpers/fetchTaxonAndSave.js

```diff
@@ -0,0 +1,23 @@
+import { fetchTaxon } from "api/taxa";
+import { getJWT } from "components/LoginSignUp/AuthenticationService.ts";
+import Taxon from "realmModels/Taxon";
+import safeRealmWrite from "sharedHelpers/safeRealmWrite";
+
+async function fetchTaxonAndSave( id, realm, params = {}, opts = {} ) {
```

Comment:
This was not imported in the MatchHeader tree so I did not migrate it to TypeScript.